example/centos/centos-9-aarch64-vhd.yaml

```diff
@@ -0,0 +1,311 @@
+            commands:
+              - command: blacklist
+                modulename: intel_cstate
+        - type: org.osbuild.modprobe
+          options:
+            filename: blacklist-floppy.conf
+            commands:
+              - command: blacklist
+                modulename: floppy
+        - type: org.osbuild.modprobe
+          options:
+            filename: blacklist-nouveau.conf
+            commands:
+              - command: blacklist
+                modulename: nouveau
+              - command: blacklist
+                modulename: lbm-nouveau
+        - type: org.osbuild.modprobe
+          options:
+            filename: blacklist-skylake-edac.conf
+            commands:
+              - command: blacklist
+                modulename: skx_edac
+        - type: org.osbuild.systemd.unit
+          options:
+            unit: nm-cloud-setup.service
+            dropin: 10-rh-enable-for-azure.conf
+            config:
+              Service:
+                Environment:
+                  - key: NM_CLOUD_SETUP_AZURE
+                    value: 'yes'
+        - type: org.osbuild.sshd.config
+          options:
+            config:
+              ClientAliveInterval: 180
+        - type: org.osbuild.pwquality.conf
+          options:
+            config:
+              minlen: 6
+              dcredit: 0
+              ucredit: 0
+              lcredit: 0
+              ocredit: 0
+              minclass: 3
+        - type: org.osbuild.waagent.conf
+          options:
+            config:
+              ResourceDisk.Format: false
+              ResourceDisk.EnableSwap: false
+        - type: org.osbuild.udev.rules
+          options:
+            filename: /etc/udev/rules.d/68-azure-sriov-nm-unmanaged.rules
+            rules:
+              - comment:
+                  - Accelerated Networking on Azure exposes a new SRIOV interface to
+                    the VM.
+                  - This interface is transparently bonded to the synthetic interface,
+                  - so NetworkManager should just ignore any SRIOV interfaces.
+              - - key: SUBSYSTEM
+                  op: ==
+                  val: net
+                - key: DRIVERS
+                  op: ==
+                  val: hv_pci
+                - key: ACTION
+                  op: ==
+                  val: add
+                - key:
+                    name: ENV
+                    arg: NM_UNMANAGED
+                  op: '='
+                  val: '1'
+        - otk.external.otk-make-fstab-stage:
+            ${filesystem}
+        - type: org.osbuild.grub2
+          options:
+            root_fs_uuid: 9851898e-0b30-437d-8fad-51ec16c3697f
+            boot_fs_uuid: dbd21911-1c4e-4107-8a9f-14fe6e751358
+            kernel_opts: ${kernel.cmdline}
+            uefi:
+              vendor: centos
+              unified: true
+            saved_entry: ffffffffffffffffffffffffffffffff-8-2.fk1.aarch64
```

Comment:
This could be `ffffffffffffffffffffffffffffffff-${kernel.package.version}-${kernel.package.release}.${kernel.package.arch}` but I don't think we are very consistent about it already so we could do that later. It would be nice to fix it (here and in the other places) because it is one of those things that will prevent otk files to work "for real".

Reply:
Done.